plugins/inputs/logfile/tail/tail_test.go

```diff
@@ -8,8 +8,12 @@ import (
 	"strings"
 	"testing"
 	"time"
+
+	"gotest.tools/assert"
 )
 
+var linesWrittenToFile int = 10
```

Comment:
Unless I skimmed this file incorrectly, this looks like it could be a const.

Reply:
I forgot const existed...
Will fix.